### DISCUSSION TITLE
[Request] HAproxy

### DISCUSSION THREAD

**Discussion by mschabhuettl** [tteck/Proxmox#323] [Ideas]
Hey!

HAproxy (https://www.haproxy.org/) as an LXC would be really cool!

Cheers!

**Comment by tteck** [comment]
Just the base haproxy (2.6) with a documented `haproxy.cfg` path so the user can configure?

**Reply by mschabhuettl** [reply]
@tteck yeah, should suffice!

**Reply by tteck** [reply]
what would be preferred? I've never ran haproxy

**Reply by mschabhuettl** [reply]
haven't used haproxy yet... but i'd love to.
base haproxy should be enough
